core/src/main/kotlin/team/backend/exception/CommonExceptionHandler.kt

```diff
@@ -0,0 +1,24 @@
+package team.backend.exception
+
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.RestControllerAdvice
+import reactor.core.publisher.Mono
+import team.backend.response.CommonResponse
+import team.backend.response.ResponseCode
+
+@RestControllerAdvice
+class CommonExceptionHandler {
+
+    @ExceptionHandler(Exception::class)
+    fun exception(e: Exception): Mono<ResponseEntity<CommonResponse<Unit>>> {
+        val errorCode = ResponseCode.INTERNAL_SERVER_ERROR
+        return Mono.just(ResponseEntity(CommonResponse.fail(errorCode), errorCode.httpStatus))
```

Comment:
It is recommended that Exception, which belongs to a higher tier, go to the bottom.

Reply:
__Do Consider: [Documents](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/)__
- As of Spring Framework 6.0, [RFC 7807 Problem Details](https://docs.spring.io/spring-framework/docs/6.0.8/reference/html/web.html#mvc-ann-rest-exceptions) is supported. Spring MVC can produce custom error messages with the application/problem+json media type, like:

```
{
  "type": "https://example.org/problems/unknown-project",
  "title": "Unknown project",
  "status": 404,
  "detail": "No project found for id 'spring-unknown'",
  "instance": "/projects/spring-unknown"
}
```

So, I think it would be good to take the success response spec and the failure response spec separately. (spec is json format)

Reply:
[Reactive stack Error Responses](https://docs.spring.io/spring-framework/docs/6.0.8/reference/html/web-reactive.html#webflux-ann-rest-exceptions)

Reply:
@BAEKJungHo 
It would be useful to divide the success and failure spec.
However, not all of the spec fields defined above seem necessary. It seems like we need to make an effort to input data that is not of significant importance or meaningful.
We need to discuss the failure response spec.

Reply:
@mangchhe 
Yes, not all properties are required. I think the `errorFields` should be acceptable in the case of Validation Exception. 

And, You can add anything else you feel you need.